### PR TITLE
delete use KeyValueStore

### PR DESCRIPTION
**Pull request by firedpeanut**
optimize code

### CODE REVIEW

File: linera-sdk/src/views/mod.rs [hunk 1]
```diff
@@ -6,7 +6,7 @@
 mod conversions_from_wit;
 mod system_api;
 
-pub use self::system_api::{KeyValueStore, ViewStorageContext};
```
Comment (ma2bd) [inline linera-sdk/src/views/mod.rs:9]: I'm supportive but you also need to fix the docs.

**Comment by firedpeanut** [comment]
is this ok?

**Review by ma2bd — CHANGES_REQUESTED** [review]
back to your queue

File: linera-sdk/src/views/system_api.rs [hunk 2]
```diff
@@ -90,6 +90,6 @@ impl KeyValueStoreClient for KeyValueStore {
     }
 }
 
-/// Implementation of [`linera_views::common::Context`] that uses the [`KeyValueStore`] to
+/// Implementation of [`linera_views::common::Context`] that uses the KeyValueStore to
```
[the remark this answers is not in the document]
Comment (firedpeanut) [reply linera-sdk/src/views/system_api.rs]: OK

**Review by ma2bd — APPROVED** [review]

**Pull request merged**